tests/framework/Optimizers/GeneticAlgorithms/testGAMaxwoRepDifferentDist.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers/GA/MaxwReplacement</name>
```

Comment:
No gold files are provided for this test.

Reply:
Gold files added